build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
Hmm. Why are you deleting this?

The usual way I upgrade is to update the version here and then do a "./gradlew wrapper" (sometimes twice? *shrug*). How did you perform the upgrade?